modules/back-end/src/Api/Api.csproj

```diff
@@ -10,7 +10,9 @@
 
     <ItemGroup>
         <!-- Api Versioning -->
-        <PackageReference Include="Asp.Versioning.Mvc.ApiExplorer" Version="6.0.0"/>
+        <PackageReference Include="Asp.Versioning.Mvc.ApiExplorer" Version="6.0.0" />
+        <PackageReference Include="Serilog.Sinks.Console" Version="4.1.0" />
```

Comment:
We needn't reference the `Serilog.Sinks.Console` package explicitly because it has been referenced by `Serilog.AspNetCore` package.